internal/file/file_helper.go

```diff
@@ -0,0 +1,92 @@
+package file
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/yaml"
+)
+
+type File struct {
+	Info         os.FileInfo
+	Unstructured unstructured.Unstructured
+}
+
+func ReadFiles(dir string) ([]File, error) {
+	log := logrus.New()
+
+	files, err := ioutil.ReadDir(dir)
+	if err != nil {
+		return nil, err
+	}
+	return readFiles(dir, files, log)
+}
+
+func readFiles(path string, files []os.FileInfo, log *logrus.Logger) ([]File, error) {
```

Comment:
I wonder if we would want to make this a walker eventually. Not totally sure though

Reply:
Maybe the recursive thing works fine for now I think, what is the benefit of a walker?

Reply:
I agree, this might be an optimization later but it works now so I think I we should just keep it and can fix in a min.